modules/ROOT/pages/message-type-source-at-host.adoc

```diff
@@ -1,8 +1,18 @@
 = Source at Host Message Type Settings
 
+:message-type: Source at Host
+
 The Source at Host message type defines the way in which your backend applications transform outbound transactions and send them to your organization.
 
-Use the following settings to define JSON or XML settings for the Source at Host message type:
+You can define the CSV, JSON, or XML formats for the Source at Host message type.
+
+== CSV
+
+include::partial$csv-outbound-partial.adoc[]
```

Comment:
Since this is "Source" - we should be showing the settings for csv-inbound-parial.adoc. All the parameters like Separator, Quotes etc must be shown in the configuration for this type.